### ISSUE TITLE
Please enable the "GitHub Discussions" feature

### ISSUE THREAD

**Issue by unforgettableid** [freeotp/freeotp-android#269]
Dear @npmccallum, dear @adambkaplan, and dear other maintainers:

There's no FreeOTP email list.  Whether or not a new version is coming soon — and even if no new version is ever released again — it still would be useful to allow users of the current version to communicate with each other.

I wonder if you could please enable the "GitHub Discussions" feature?

It's quite easy to do.  If you wish to enable the feature, you simply:

* Go to https://github.com/freeotp/freeotp-android/settings.
* Scroll down.
* Click the green "Set up discussions" button.
* Scroll down again.
* Click the green "Start discussion" button.

Any thoughts?

Thank you for reading this!

**Comment by wmealing** [comment]
Not project management of this project, but this can lead to admin overhead that they may not want at this time, it can get a bit messy and time consuming to maintain.

**Comment by unforgettableid** [comment]
That problem might not happen. And, even if it does happen, they can just disable the Discussions feature again, if they wish.

**Comment by jimmysjolund** [comment]
I agree there are good reasons to activate Discussions. As I have been going through all open issues this week, many of them would have been a better fit for Discussion rather than issue. As mentioned in 2023, worst case, it can be deactivated again.
